src/builtin-addons/core/intl-utils.ts

```diff
@@ -30,20 +35,43 @@ type TranslationFile =
 
 const YAML_EXTENSIONS = ['.yaml', '.yml'];
 
+export function getEmberIntlConfig(root: string): EmberIntlConfig | null {
+  try {
+    if (!getRequireSupport()) {
+      return null;
+    }
+
+    // @ts-expect-error @todo - fix webpack imports
+    const requireFunc = typeof __webpack_require__ === 'function' ? __non_webpack_require__ : require;
+
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const intlConfig = requireFunc(path.join(root, 'config', 'ember-intl.js'));
+
+    if (!intlConfig) {
+      return null;
+    }
+
+    return typeof intlConfig === 'function' ? intlConfig() : intlConfig;
```

Comment:
@lifeart This is perplexing me a bit as to how I should treat this `intlConfig`. Essentially I want to return the `IntlConfig` back as an object here. Debugging `jest` is causing me a bit of pain to figure it out 😅

My debugger is telling me this is returned as function but even when I add `intlConfig()` it's still returned as a function 😬 Is there something obvious I'm missing here?

```js
config: {
  'ember-intl.js': `module.exports = function() { return { wrapTranslationsWithNamespace: true } }`,
},
```

Reply:
Wow, it may be jest problem, if it's acting as a function, try to `toString()` it to figure out what is it.
In general, if tests is passing, it should work fine.
Provided config is correct.